checker/jtreg/nullness/AssignmentPerformanceTest.java

```diff
@@ -1,6 +1,9 @@
-// @skip-test
-// Test case for issue #536: https://github.com/typetools/checker-framework/issues/536
-
+/*
+ * @test
+ * @summary Test case for issue #536: https://github.com/typetools/checker-framework/issues/536
+ *
+ * @compile/timeout=60 -J-Xmx256M -XDrawDiagnostics -Xlint:unchecked -processor org.checkerframework.checker.nullness.NullnessChecker -AprintErrorStack -Alint AssignmentPerformanceTest.java
```

Comment:
The `@test` style only works for jtreg style tests. You would need to move the test to a different directory for this.
I'm not sure we need to do this, so I would undo these changes and just remove the `@skip-test`.